pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaExecutionResult.java

```diff
@@ -29,6 +29,7 @@
 public class JavaExecutionResult {
     private Throwable userException;
     private Object result;
+    private double startTimeNanos;
```

Comment:
```suggestion
    private final long startTimeNanos = System.nanoTime();
```
It should be a `long` field and it should be made final.